src/app/containers/Footer/__snapshots__/index.test.jsx.snap

```diff
@@ -6,7 +6,7 @@ exports[`FooterContainer should render correctly 1`] = `
 >
   <BrandContainer />
   <SitewideLinks
-    copyrightText="© 2018 BBC. The BBC is not responsible for the content of external sites. "
+    copyrightText="© 2019 BBC. The BBC is not responsible for the content of external sites. "
```

Comment:
It may only be once a year, but this is another temporary fix. 🤔 Could we mock out `Date.getFullYear()` in the test file?